dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -1173,6 +1173,6 @@ public int countNotArchivedItems(Context context) throws SQLException {
     @Override
     public int countWithdrawnItems(Context context) throws SQLException {
        // return count of items that are in archive and withdrawn
```

Comment:
On Thu, 25 Feb 2016 07:16:33 -0800
Pascal-Nicolas Becker <notifications@github.com> wrote:

> > @@ -1173,6 +1173,6 @@ public int countNotArchivedItems(Context
> > context) throws SQLException { @Override
> >      public int countWithdrawnItems(Context context) throws
> > SQLException { // return count of items that are in archive and
> > withdrawn
> 
> Can you pleas also change this comment?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/DSpace/DSpace/pull/1320/files#r54109668

Sure.  I have just done that.  Sorry for not looking into comments more
carefully.

-- 
Mateusz